client/components/post-excerpt/index.jsx

```diff
@@ -1,31 +1,33 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classnames = require( 'classnames' );
+import React from 'react';
+import classnames from 'classnames';
+import striptags from 'striptags';
 
-var PostExcerpt = React.createClass( {
+const PostExcerpt = React.createClass( {
 
-	render: function() {
-		var text = this.props.text,
-			textClass = [ 'post-excerpt__text' ];
+	propTypes: {
+		content: React.PropTypes.string.isRequired
+	},
 
-		if ( ! text ) {
+	render() {
+		if ( ! this.props.content ) {
 			return null;
 		}
 
-		if ( text.length > 80 ) {
-			textClass.push( 'is-long' );
-		}
+		// Strip all non-linebreak tags
+		const preparedContent = striptags( this.props.content, [ 'p', 'br' ] );
```

Comment:
let's to this as a part of state. otherwise we're stripping the content on every single render call